src/cloud/index.ts

```diff
@@ -9,7 +9,7 @@ interface ConnectData {
   endpoint?: string;
 }
 
-const SYNC_DB_NAME = "_fp.sync";
+const SYNC_DB_NAME = "fp_sync";
```

Comment:
I like it, I'll merge now but I assume there will be a parallel change on the dashboard side